authenticity/e-check-diagnose.yml

```diff
@@ -226,4 +227,5 @@ components:
         - "HOLO_PHOTO_DOCUMENT_OUTSIDE_FRAME"
         - "LIVENESS_DEPTH_CHECK_FAILED"
         - "LAS_INK_INVALID_LINES_FREQUENCY"
-        - "LAST_DIAGNOSE_VALUE"
```

Comment:
wrong

Reply:
Fixed in https://github.com/regulaforensics/DocumentReader-web-openapi/pull/108